code/game/objects/items/tools/crowbar.dm

```diff
@@ -3,6 +3,8 @@
 	desc = "A small crowbar. This handy tool is useful for lots of things, such as prying floor tiles or opening unpowered doors."
 	icon = 'icons/obj/tools.dmi'
 	icon_state = "crowbar"
+	worn_icon_state = "crowbar"
+	worn_icon = 'icons/mob/clothing/belt.dmi'

```

Comment:
handled by build_worn_icon() already. this will use the incorrect icon if put in a suit slot.

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.